### ISSUE TITLE
Logging the model's parameters - initialize ´wblogger´

### ISSUE THREAD

**Issue by saraalrawi** [avik-pal/Wandb.jl#2]
Hello everyone, 

Thanks for the nice work. 
Could you please tell me where is  ´wblogger´ defined in the following line?

´log(wblogger, cpu; parameters = ps, gradients = gs, commit = false)´

Thanks in advance!

**Comment by avik-pal** [comment]
Thanks for pointing it out. `wblogger` should be something like `WandbLogger(
    project = "Wandb.jl",
    name = "fluxjl-integration-$(now())",
    config = Dict(
        "learning_rate" => 3e-4,
        "batchsize" => 256,
        "epochs" => 100,
        "dataset" => "MNIST",
        "use_cuda" => true,
    ),
)`. But I removed that particular function dispatch in the current release (I haven't updated the docs). The current way to do it would be to log the histograms manually.
